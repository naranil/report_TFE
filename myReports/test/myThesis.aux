\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\bibstyle{plain}
\@newglossary{main}{glg}{gls}{glo}
\@istfilename{myThesis.xdy}
\@glsorder{word}
\@writefile{toc}{\contentsline {chapter}{Introduction}{1}{chapter*.7}}
\citation{freeley2000argumentation}
\citation{freeley2000argumentation}
\citation{freeley2000argumentation}
\citation{freeley2000argumentation}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Background}{2}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Argumentation}{2}{section.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}General Definitions}{2}{subsection.1.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.2}Persuasion}{2}{subsection.1.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.3}Argumentation Mining}{3}{subsection.1.1.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Short caption}}{4}{figure.1.1}}
\newlabel{fig:forumspam}{{1.1}{4}{Short caption}{figure.1.1}{}}
\newlabel{fig:forumspam@cref}{{[figure][1][1]1.1}{4}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.2}{\ignorespaces Short caption}}{4}{figure.1.2}}
\newlabel{fig:argmining}{{1.2}{4}{Short caption}{figure.1.2}{}}
\newlabel{fig:argmining@cref}{{[figure][2][1]1.2}{4}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}NLP and the DKPro Framework}{4}{section.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.1}Natural Language Processing}{4}{subsection.1.2.1}}
\citation{GurevychEtal2007dkpro0}
\citation{uima:Online}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.2}UIMA}{5}{subsection.1.2.2}}
\citation{TUD-CS-2014-0864}
\@writefile{lof}{\contentsline {figure}{\numberline {1.3}{\ignorespaces Short caption}}{6}{figure.1.3}}
\newlabel{fig:uima}{{1.3}{6}{Short caption}{figure.1.3}{}}
\newlabel{fig:uima@cref}{{[figure][3][1]1.3}{6}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.3}DKPro Core}{6}{subsection.1.2.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.4}{\ignorespaces Short caption}}{7}{figure.1.4}}
\newlabel{fig:dkpro-pipeline}{{1.4}{7}{Short caption}{figure.1.4}{}}
\newlabel{fig:dkpro-pipeline@cref}{{[figure][4][1]1.4}{7}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2.4}DKPro Text Classification}{7}{subsection.1.2.4}}
\@writefile{lof}{\contentsline {figure}{\numberline {1.5}{\ignorespaces Short caption}}{8}{figure.1.5}}
\newlabel{fig:dkpro-tc-usages}{{1.5}{8}{Short caption}{figure.1.5}{}}
\newlabel{fig:dkpro-tc-usages@cref}{{[figure][5][1]1.5}{8}}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}The Research Work}{10}{chapter.2}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {2.1}Text Corpus}{10}{section.2.1}}
\citation{ivanguideline}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.1}Manual Annotation}{11}{subsection.2.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1.1}Sources of the data}{11}{subsubsection.2.1.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1.2}Categories in Persuasion}{11}{subsubsection.2.1.1.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1.3}Examples}{12}{subsubsection.2.1.1.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1.4}Annotation Process}{13}{subsubsection.2.1.1.4}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1.5}Results of the Manual Annotations}{14}{subsubsection.2.1.1.5}}
\@writefile{lot}{\contentsline {table}{\numberline {2.1}{\ignorespaces  Human performance on \emph  {gold data persuasive}.}}{14}{table.2.1}}
\newlabel{tab:human-performance-annotation-study-1}{{2.1}{14}{Human performance on \emph {gold data persuasive}}{table.2.1}{}}
\newlabel{tab:human-performance-annotation-study-1@cref}{{[table][1][2]2.1}{14}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.2}Corpus Statistics}{14}{subsection.2.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1.3}Conflict Annotations}{15}{subsection.2.1.3}}
\newlabel{sec:conflictannotations}{{2.1.3}{15}{Conflict Annotations}{subsection.2.1.3}{}}
\newlabel{sec:conflictannotations@cref}{{[subsection][3][2,1]2.1.3}{15}}
\@writefile{toc}{\contentsline {section}{\numberline {2.2}Feature Engineering}{15}{section.2.2}}
\citation{tcjullmann77}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2.1}Lexical Features}{16}{subsection.2.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1.1}Tokens N-Grams}{16}{subsubsection.2.2.1.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.1}{\ignorespaces Short caption}}{17}{figure.2.1}}
\newlabel{fig:ngramfeature}{{2.1}{17}{Short caption}{figure.2.1}{}}
\newlabel{fig:ngramfeature@cref}{{[figure][1][2]2.1}{17}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1.2}Tokens and Sentences}{17}{subsubsection.2.2.1.2}}
\citation{Hasan-Ng:IJCNLP:2013}
\citation{anand-EtAl:2011:WASSA2011}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1.3}Other Lexical Statistics}{18}{subsubsection.2.2.1.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1.4}Punctuation Related Features}{18}{subsubsection.2.2.1.4}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.2}{\ignorespaces  6 punctuation marks}}{18}{figure.2.2}}
\newlabel{punctuation}{{2.2}{18}{6 punctuation marks}{figure.2.2}{}}
\newlabel{punctuation@cref}{{[figure][2][2]2.2}{18}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.3}{\ignorespaces Short caption}}{19}{figure.2.3}}
\newlabel{fig:multipunc}{{2.3}{19}{Short caption}{figure.2.3}{}}
\newlabel{fig:multipunc@cref}{{[figure][3][2]2.3}{19}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.1.5}Multiple Capital Letters}{19}{subsubsection.2.2.1.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2.2}Part Of Speech Features}{19}{subsection.2.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.2.1}Ratio on common POS}{19}{subsubsection.2.2.2.1}}
\@writefile{lot}{\contentsline {table}{\numberline {2.2}{\ignorespaces  The 11 POS we consider for the Ratio POS feature}}{20}{table.2.2}}
\newlabel{11pos}{{2.2}{20}{The 11 POS we consider for the Ratio POS feature}{table.2.2}{}}
\newlabel{11pos@cref}{{[table][2][2]2.2}{20}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.4}{\ignorespaces Short caption}}{20}{figure.2.4}}
\newlabel{fig:11poscode}{{2.4}{20}{Short caption}{figure.2.4}{}}
\newlabel{fig:11poscode@cref}{{[figure][4][2]2.4}{20}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.2.2}Comparative and Superlative}{20}{subsubsection.2.2.2.2}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.2.3}Modal Verbs}{20}{subsubsection.2.2.2.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.2.4}POS N-Grams}{20}{subsubsection.2.2.2.4}}
\citation{Ã¡gel2006dependency}
\citation{TUD-CS-2014-0882}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2.3}Syntactic features}{21}{subsection.2.2.3}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.3.1}Depth of the Dependency Tree}{21}{subsubsection.2.2.3.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.5}{\ignorespaces Short caption}}{21}{figure.2.5}}
\newlabel{fig:deptree}{{2.5}{21}{Short caption}{figure.2.5}{}}
\newlabel{fig:deptree@cref}{{[figure][5][2]2.5}{21}}
\citation{Nivre_aquick}
\citation{TUD-CS-2014-0882}
\@writefile{lof}{\contentsline {figure}{\numberline {2.6}{\ignorespaces Short caption}}{22}{figure.2.6}}
\newlabel{fig:deptreepath}{{2.6}{22}{Short caption}{figure.2.6}{}}
\newlabel{fig:deptreepath@cref}{{[figure][6][2]2.6}{22}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.7}{\ignorespaces  MaltParser's output tree}}{22}{figure.2.7}}
\newlabel{dkprotree}{{2.7}{22}{MaltParser's output tree}{figure.2.7}{}}
\newlabel{dkprotree@cref}{{[figure][7][2]2.7}{22}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.3.2}Dependency Rules}{22}{subsubsection.2.2.3.2}}
\citation{TUD-CS-2014-0882}
\citation{Socher_recursivedeep}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.3.3}Subordinate clauses}{23}{subsubsection.2.2.3.3}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.8}{\ignorespaces Short caption}}{23}{figure.2.8}}
\newlabel{fig:subclause}{{2.8}{23}{Short caption}{figure.2.8}{}}
\newlabel{fig:subclause@cref}{{[figure][8][2]2.8}{23}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2.4}Sentiment Analysis Feature}{23}{subsection.2.2.4}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.9}{\ignorespaces Short caption}}{24}{figure.2.9}}
\newlabel{fig:sentcoeff}{{2.9}{24}{Short caption}{figure.2.9}{}}
\newlabel{fig:sentcoeff@cref}{{[figure][9][2]2.9}{24}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.4.1}Sentiment Coefficients}{24}{subsubsection.2.2.4.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.2.4.2}Sentiment Fluctuation}{25}{subsubsection.2.2.4.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2.5}LDA}{25}{subsection.2.2.5}}
\citation{Blei:2003:LDA:944919.944937}
\citation{Boser:1992}
\@writefile{toc}{\contentsline {section}{\numberline {2.3}The Classifier and Performances}{26}{section.2.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3.1}SVM Classifier}{26}{subsection.2.3.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.10}{\ignorespaces Short caption}}{26}{figure.2.10}}
\newlabel{fig:svm-nonlin}{{2.10}{26}{Short caption}{figure.2.10}{}}
\newlabel{fig:svm-nonlin@cref}{{[figure][10][2]2.10}{26}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.11}{\ignorespaces Short caption}}{27}{figure.2.11}}
\newlabel{fig:svm-lin}{{2.11}{27}{Short caption}{figure.2.11}{}}
\newlabel{fig:svm-lin@cref}{{[figure][11][2]2.11}{27}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.12}{\ignorespaces Short caption}}{28}{figure.2.12}}
\newlabel{fig:hypopt}{{2.12}{28}{Short caption}{figure.2.12}{}}
\newlabel{fig:hypopt@cref}{{[figure][12][2]2.12}{28}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.13}{\ignorespaces Short caption}}{29}{figure.2.13}}
\newlabel{fig:svm-slack}{{2.13}{29}{Short caption}{figure.2.13}{}}
\newlabel{fig:svm-slack@cref}{{[figure][13][2]2.13}{29}}
\@writefile{lof}{\contentsline {figure}{\numberline {2.14}{\ignorespaces Short caption}}{29}{figure.2.14}}
\newlabel{fig:svm-C}{{2.14}{29}{Short caption}{figure.2.14}{}}
\newlabel{fig:svm-C@cref}{{[figure][14][2]2.14}{29}}
\citation{Platt:1999}
\@writefile{lof}{\contentsline {figure}{\numberline {2.15}{\ignorespaces Short caption}}{30}{figure.2.15}}
\newlabel{fig:svm-kernel}{{2.15}{30}{Short caption}{figure.2.15}{}}
\newlabel{fig:svm-kernel@cref}{{[figure][15][2]2.15}{30}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3.2}Weka's SMO Classifier}{30}{subsection.2.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3.3}Evaluate the performances of our classification}{30}{subsection.2.3.3}}
\@writefile{toc}{\contentsline {chapter}{\numberline {3}Results and Perspective}{32}{chapter.3}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {3.1}Performance Evaluation}{32}{section.3.1}}
\@writefile{toc}{\contentsline {section}{\numberline {3.2}Feature Selection}{34}{section.3.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3.3}Error Analysis}{36}{section.3.3}}
\citation{NLE:9163968}
\@writefile{toc}{\contentsline {section}{\numberline {3.4}Future Work and General Conclusion}{38}{section.3.4}}
\@writefile{toc}{\contentsline {section}{\numberline {3.5}Personal Conclusion}{39}{section.3.5}}
\@writefile{toc}{\contentsline {chapter}{\numberline {A}Work Packages}{40}{appendix.A}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{sec:schedule}{{A}{40}{Work Packages}{appendix.A}{}}
\newlabel{sec:schedule@cref}{{[appendix][1][2147483647]A}{40}}
\@writefile{lof}{\contentsline {figure}{\numberline {A.1}{\ignorespaces Short caption}}{40}{figure.A.1}}
\newlabel{fig:schedule}{{A.1}{40}{Short caption}{figure.A.1}{}}
\newlabel{fig:schedule@cref}{{[figure][1][2147483647,1]A.1}{40}}
\@writefile{toc}{\contentsline {chapter}{\numberline {B}DKPro Core Overview}{41}{appendix.B}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{lof}{\contentsline {figure}{\numberline {B.1}{\ignorespaces Short caption}}{41}{figure.B.1}}
\newlabel{fig:dkpro-overview}{{B.1}{41}{Short caption}{figure.B.1}{}}
\newlabel{fig:dkpro-overview@cref}{{[figure][1][2147483647,2]B.1}{41}}
\newlabel{sec:dkpro overview}{{B}{41}{DKPro Core Overview}{figure.B.1}{}}
\newlabel{sec:dkpro overview@cref}{{[appendix][2][2147483647]B}{41}}
\@writefile{toc}{\contentsline {chapter}{\numberline {C}Xmi Files - Example}{42}{appendix.C}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{sec:xmi}{{C}{42}{Xmi Files - Example}{appendix.C}{}}
\newlabel{sec:xmi@cref}{{[appendix][3][2147483647]C}{42}}
\@writefile{lof}{\contentsline {figure}{\numberline {C.1}{\ignorespaces Short caption}}{42}{figure.C.1}}
\newlabel{fig:dkpro-overview}{{C.1}{42}{Short caption}{figure.C.1}{}}
\newlabel{fig:dkpro-overview@cref}{{[figure][1][2147483647,3]C.1}{42}}
\@writefile{toc}{\contentsline {chapter}{\numberline {D}Binary Classification}{43}{appendix.D}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{sec:bincla}{{D}{43}{Binary Classification}{appendix.D}{}}
\newlabel{sec:bincla@cref}{{[appendix][4][2147483647]D}{43}}
\@writefile{toc}{\contentsline {chapter}{\numberline {E}Error Analysis - Confusion Matrices}{45}{appendix.E}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{sec:errana}{{E}{45}{Error Analysis - Confusion Matrices}{appendix.E}{}}
\newlabel{sec:errana@cref}{{[appendix][5][2147483647]E}{45}}
\bibdata{bib/Bibliography}
\bibcite{uima:Online}{1}
\bibcite{Ã¡gel2006dependency}{2}
\bibcite{anand-EtAl:2011:WASSA2011}{3}
\bibcite{Blei:2003:LDA:944919.944937}{4}
\bibcite{Boser:1992}{5}
\bibcite{TUD-CS-2014-0864}{6}
\bibcite{freeley2000argumentation}{7}
\bibcite{GurevychEtal2007dkpro0}{8}
\bibcite{ivanguideline}{9}
\bibcite{Hasan-Ng:IJCNLP:2013}{10}
\bibcite{NLE:9163968}{11}
\bibcite{Nivre_aquick}{12}
\bibcite{Platt:1999}{13}
\bibcite{Socher_recursivedeep}{14}
\bibcite{TUD-CS-2014-0882}{15}
\bibcite{tcjullmann77}{16}
\@xdylanguage{main}{english}
\@gls@codepage{main}{}
