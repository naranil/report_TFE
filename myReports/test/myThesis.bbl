\begin{thebibliography}{1}

\bibitem{uima:Online}
Overview and setup, uima.apache.org, 2006.

\bibitem{Ã¡gel2006dependency}
V.~{\'A}gel.
\newblock {\em Dependency and valency: an international handbook of
  contemporary research}.
\newblock Dependenz und Valenz: ein internationales Handbuch der
  zeitgen{\"o}ssischen Forschung. de Gruyter, 2006.

\bibitem{anand-EtAl:2011:WASSA2011}
Pranav Anand, Marilyn Walker, Rob Abbott, Jean~E. Fox~Tree, Robeson Bowmani,
  and Michael Minor.
\newblock {Cats Rule and Dogs Drool!: Classifying Stance in Online Debate}.
\newblock In {\em Proceedings of the 2nd Workshop on Computational Approaches
  to Subjectivity and Sentiment Analysis (WASSA 2.011)}, pages 1--9, Portland,
  Oregon, June 2011. Association for Computational Linguistics.

\bibitem{TUD-CS-2014-0864}
Richard~Eckart de~Castilho and Iryna Gurevych.
\newblock A broad-coverage collection of portable nlp components for building
  shareable analysis pipelines.
\newblock In {\em Proceedings of the Workshop on Open Infrastructures and
  Analysis Frameworks for HLT (OIAF4HLT) at COLING 2014}, pages 1--11, August
  2014.

\bibitem{GurevychEtal2007dkpro0}
Iryna Gurevych, Max M{\"u}hlh{\"a}user, Christof M{\"u}ller, J{\"u}rgen
  Steimle, Markus Weimer, and Torsten Zesch.
\newblock Darmstadt knowledge processing repository based on uima.
\newblock In {\em Proceedings of the First Workshop on Unstructured Information
  Management Architecture at Biannual Conference of the Society for
  Computational Linguistics and Language Technology}, T{\"u}bingen, Germany,
  April 2007.

\bibitem{ivanguideline}
Ivan Habernal, Judith Eckle-Kohler, and Iryna Gurevych.
\newblock Argumentation mining on the web from information seeking perspective.
\newblock In {\em Frontiers and Connections between Argumentation Theory and
  Natural Language Processing}, page (to appear), July 2014.

\bibitem{Hasan-Ng:IJCNLP:2013}
Kazi~Saidul Hasan and Vincent Ng.
\newblock Stance classification of ideological debates: Data, models, features,
  and constraints.
\newblock In {\em Proceedings of the Sixth International Joint Conference on
  Natural Language Processing}, pages 1348--1356, 2013.

\bibitem{TUD-CS-2014-0882}
Christian Stab and Iryna Gurevych.
\newblock Identifying argumentative discourse structures in persuasive essays.
\newblock In {\em Conference on Empirical Methods in Natural Language
  Processing (EMNLP 2014)}, page (to appear), Stroudsburg, PA, USA, October
  2014. Association for Computational Linguistics.

\bibitem{tcjullmann77}
Julian~R. Ullmann.
\newblock A binary $n$-gram technique for automatic correction of substitution,
  deletion, insertion and reversal errors in words.
\newblock {\em The Computer Journal}, 20(2):141--147, 1977.

\end{thebibliography}
