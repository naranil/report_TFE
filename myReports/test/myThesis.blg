This is BibTeX, Version 0.99d (TeX Live 2013/Debian)
Capacity: max_strings=35307, hash_size=35307, hash_prime=30011
The top-level auxiliary file: myThesis.aux
The style file: plain.bst
Database file #1: bib/Bibliography.bib
Warning--entry type for "uima:Online" isn't style-file defined
--line 220 of file bib/Bibliography.bib
Warning--to sort, need author or key in uima:Online
Warning--empty booktitle in Platt:1999
You've used 16 entries,
            2118 wiz_defined-function locations,
            602 strings with 7645 characters,
and the built_in function-call counts, 6115 in all, are:
= -- 619
> -- 263
< -- 8
+ -- 109
- -- 88
* -- 375
:= -- 923
add.period$ -- 50
call.type$ -- 16
change.case$ -- 90
chr.to.int$ -- 0
cite$ -- 18
duplicate$ -- 276
empty$ -- 489
format.name$ -- 88
if$ -- 1362
int.to.chr$ -- 0
int.to.str$ -- 16
missing$ -- 15
newline$ -- 79
num.names$ -- 30
pop$ -- 126
preamble$ -- 1
purify$ -- 76
quote$ -- 0
skip$ -- 234
stack$ -- 0
substring$ -- 346
swap$ -- 104
text.length$ -- 8
text.prefix$ -- 0
top$ -- 0
type$ -- 60
warning$ -- 2
while$ -- 53
width$ -- 18
write$ -- 173
(There were 3 warnings)
