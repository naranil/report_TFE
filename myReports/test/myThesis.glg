;; This logfile was generated automatically by `xindy'
;; at 09.09.2014  17:49:46
;; Indexstyle: "/tmp/rQqjUxiXEa", Rawindex: "/tmp/WxlOW7va8y", Output: "myThesis.gls"


Building indexentry-tree:
Hierdepth of index is :TREE

sort-indexentries: (FORWARD BACKWARD FORWARD FORWARD)

Forming letter-groups:
sort-indexentries: (FORWARD BACKWARD FORWARD FORWARD)
Letter-group: "»€Ú¤—»€Ú¨†Ð" -> "M"