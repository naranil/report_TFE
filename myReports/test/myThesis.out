\BOOKMARK [0][-]{chapter*.7}{Introduction}{}% 1
\BOOKMARK [0][-]{chapter.1}{Background}{}% 2
\BOOKMARK [1][-]{section.1.1}{Argumentation}{chapter.1}% 3
\BOOKMARK [2][-]{subsection.1.1.1}{General Definitions}{section.1.1}% 4
\BOOKMARK [2][-]{subsection.1.1.2}{Persuasion}{section.1.1}% 5
\BOOKMARK [2][-]{subsection.1.1.3}{Argumentation Mining}{section.1.1}% 6
\BOOKMARK [1][-]{section.1.2}{NLP and the DKPro Framework}{chapter.1}% 7
\BOOKMARK [2][-]{subsection.1.2.1}{Natural Language Processing}{section.1.2}% 8
\BOOKMARK [2][-]{subsection.1.2.2}{UIMA}{section.1.2}% 9
\BOOKMARK [2][-]{subsection.1.2.3}{DKPro Core}{section.1.2}% 10
\BOOKMARK [2][-]{subsection.1.2.4}{DKPro Text Classification}{section.1.2}% 11
\BOOKMARK [0][-]{chapter.2}{The Research Work}{}% 12
\BOOKMARK [1][-]{section.2.1}{Text Corpus}{chapter.2}% 13
\BOOKMARK [2][-]{subsection.2.1.1}{Manual Annotation}{section.2.1}% 14
\BOOKMARK [3][-]{subsubsection.2.1.1.1}{Sources of the data}{subsection.2.1.1}% 15
\BOOKMARK [3][-]{subsubsection.2.1.1.2}{Categories in Persuasion}{subsection.2.1.1}% 16
\BOOKMARK [3][-]{subsubsection.2.1.1.3}{Examples}{subsection.2.1.1}% 17
\BOOKMARK [3][-]{subsubsection.2.1.1.4}{Annotation Process}{subsection.2.1.1}% 18
\BOOKMARK [3][-]{subsubsection.2.1.1.5}{Results of the Manual Annotations}{subsection.2.1.1}% 19
\BOOKMARK [2][-]{subsection.2.1.2}{Corpus Statistics}{section.2.1}% 20
\BOOKMARK [2][-]{subsection.2.1.3}{Conflict Annotations}{section.2.1}% 21
\BOOKMARK [1][-]{section.2.2}{Feature Engineering}{chapter.2}% 22
\BOOKMARK [2][-]{subsection.2.2.1}{Lexical Features}{section.2.2}% 23
\BOOKMARK [3][-]{subsubsection.2.2.1.1}{Tokens N-Grams}{subsection.2.2.1}% 24
\BOOKMARK [3][-]{subsubsection.2.2.1.2}{Tokens and Sentences}{subsection.2.2.1}% 25
\BOOKMARK [3][-]{subsubsection.2.2.1.3}{Other Lexical Statistics}{subsection.2.2.1}% 26
\BOOKMARK [3][-]{subsubsection.2.2.1.4}{Punctuation Related Features}{subsection.2.2.1}% 27
\BOOKMARK [3][-]{subsubsection.2.2.1.5}{Multiple Capital Letters}{subsection.2.2.1}% 28
\BOOKMARK [2][-]{subsection.2.2.2}{Part Of Speech Features}{section.2.2}% 29
\BOOKMARK [3][-]{subsubsection.2.2.2.1}{Ratio on common POS}{subsection.2.2.2}% 30
\BOOKMARK [3][-]{subsubsection.2.2.2.2}{Comparative and Superlative}{subsection.2.2.2}% 31
\BOOKMARK [3][-]{subsubsection.2.2.2.3}{Modal Verbs}{subsection.2.2.2}% 32
\BOOKMARK [3][-]{subsubsection.2.2.2.4}{POS N-Grams}{subsection.2.2.2}% 33
\BOOKMARK [2][-]{subsection.2.2.3}{Syntactic features}{section.2.2}% 34
\BOOKMARK [3][-]{subsubsection.2.2.3.1}{Depth of the Dependency Tree}{subsection.2.2.3}% 35
\BOOKMARK [3][-]{subsubsection.2.2.3.2}{Dependency Rules}{subsection.2.2.3}% 36
\BOOKMARK [3][-]{subsubsection.2.2.3.3}{Subordinate clauses}{subsection.2.2.3}% 37
\BOOKMARK [2][-]{subsection.2.2.4}{Sentiment Analysis Feature}{section.2.2}% 38
\BOOKMARK [3][-]{subsubsection.2.2.4.1}{Sentiment Coefficients}{subsection.2.2.4}% 39
\BOOKMARK [3][-]{subsubsection.2.2.4.2}{Sentiment Fluctuation}{subsection.2.2.4}% 40
\BOOKMARK [2][-]{subsection.2.2.5}{LDA}{section.2.2}% 41
\BOOKMARK [1][-]{section.2.3}{The Classifier and Performances}{chapter.2}% 42
\BOOKMARK [2][-]{subsection.2.3.1}{SVM Classifier}{section.2.3}% 43
\BOOKMARK [2][-]{subsection.2.3.2}{Weka's SMO Classifier}{section.2.3}% 44
\BOOKMARK [2][-]{subsection.2.3.3}{Evaluate the performances of our classification}{section.2.3}% 45
\BOOKMARK [0][-]{chapter.3}{Results and Perspective}{}% 46
\BOOKMARK [1][-]{section.3.1}{Performance Evaluation}{chapter.3}% 47
\BOOKMARK [1][-]{section.3.2}{Feature Selection}{chapter.3}% 48
\BOOKMARK [1][-]{section.3.3}{Error Analysis}{chapter.3}% 49
\BOOKMARK [1][-]{section.3.4}{Future Work and General Conclusion}{chapter.3}% 50
\BOOKMARK [1][-]{section.3.5}{Personal Conclusion}{chapter.3}% 51
\BOOKMARK [0][-]{appendix.A}{Work Packages}{}% 52
\BOOKMARK [0][-]{appendix.B}{DKPro Core Overview}{}% 53
\BOOKMARK [0][-]{appendix.C}{Xmi Files - Example}{}% 54
\BOOKMARK [0][-]{appendix.D}{Binary Classification}{}% 55
\BOOKMARK [0][-]{appendix.E}{Error Analysis - Confusion Matrices}{}% 56
